### PR TITLE
Release v4.1.3

### PR DESCRIPTION
**Pull request by tron-highfive**

### CODE REVIEW

**Review by mary-doc — APPROVED** [review]

**Review by mary-doc — APPROVED** [review]

**Review by mary-doc — APPROVED** [review]

**Comment by RonSherfey** [comment]
Update release3

**Review by muhammadhaffizz — APPROVED** [review]

**Comment by alm378** [comment]
Update

**Review by RonSherfey — CHANGES_REQUESTED** [review]
Requesting changes 1

**Review by RonSherfey — APPROVED** [review]
approved changes

**Review by RonSherfey — APPROVED** [review]

**Review by RonSherfey — APPROVED** [review]
approved changes

File: docs/releases/index.md [hunk 1]
```diff
@@ -38,26 +38,42 @@ node -e 'console.log(require("tronweb").Trx.verifySignature(
 ```
 
 ### Version Signature
-- Odyssey-3.7 
+- Odyssey-3.7
 ```shell
```
Comment (piotrm89) [inline docs/releases/index.md:42]:
```suggestion
```shell
```

**Review by piotrm89 — APPROVED** [review]

**Review by RonSherfey — APPROVED** [review]

**Pull request closed without merging**